packages/server/graphql/mutations/helpers/sendAccountRemovedToSegment.ts

```diff
@@ -0,0 +1,38 @@
+import ServerAuthToken from '../../../database/types/ServerAuthToken'
+import segmentIo from '../../../utils/segmentIo'
+
+const executeGraphQL = require('../../executeGraphQL').default
```

Comment:
-2 It's this line which causes the app to not start. It creates an import loop which blocks itself. Move it inside the function (as it was before) and it works.

Reply:
Thanks so much for this!!!